@here/harp-mapview/lib/Tile.ts

```diff
@@ -1014,6 +1021,14 @@ export class Tile implements CachedResource {
         return this.projection.worldExtent(0, 0).max.x * this.offset;
     }
 
+    /**
+     * Adds the supplied mesh to the list of three.js objects
+     * @param plane Plane to add to the [[Tile]].
+     */
+    set backgroundPlane(plane: THREE.Mesh) {
```

Comment:
Please remove this setter b/c it's really confusing, i.e. calling it multiple times will add the object multiple times instead of setting the plane